ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1143,17 +1372,30 @@ export const RuntimePanel = fp.flow(
               {getWarningMessageContent()}
             </WarningMessage>
            }
-           <FlexRow style={{justifyContent: 'space-between', marginTop: '.75rem'}}>
-             <Link
-               style={{...styles.deleteLink, ...(
-                 (disableControls || !runtimeExists) ?
-                 {color: colorWithWhiteness(colors.dark, .4)} : {}
-               )}}
-               aria-label='Delete Environment'
-               disabled={disableControls || !runtimeExists}
-               onClick={() => setPanelContent(PanelContent.Delete)}>Delete Environment</Link>
-               {!runtimeExists ? renderCreateButton() : renderNextButton()}
-           </FlexRow>
+        {enablePD && selectedCompute === ComputeType.Standard && !runtimeExists && pdExists ?
```

Comment:
Why does the selected compute type need to be Standard to show this?

Reply:
Because I'm thinking we shouldn't see anything related to PD when we are on the Dataproc panel.

Reply:
I would rather show it. However possible, I want to avoid hiding the fact that there is a live disk that is still costing money to the user.

Reply:
agree, now the delete env options will always contain the option to delete an unattached PD